packages/composite-checkout/test/lib/join-classes.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Internal dependencies
+ */
+import joinClasses from '../../src/lib/join-classes';
+
+describe( 'joinClasses', function() {
+	it( 'Empty array', function() {
```

Comment:
You may want to word these tests so that they read more like an english sentence: "it returns an empty string when passed an empty array"